docs/getting-started.md

```diff
@@ -8,6 +8,79 @@ Welcome to the Polkadot Wiki! This central source of truth contains guides for i
 want to build on Polkadot), and Maintain (for people that want to maintain the network).
 
+## What is Polkadot?
+
+Polkadot enables scalability by allowing specialized blockchains to communicate with each other in a
+secure, trust-free environment.
+
+Polkadot is built to connect and secure unique blockchains, whether they be public, permission-less
+networks, private consortium chains, or oracles and other Web3 technologies. It enables an internet
+where independent blockchains can exchange information under common security guarantees.
+
+Polkadot is a living network with the core pillars of governance and upgradability. The network has
+an advanced suite of governance tools and, using the WebAssembly standard as a "meta-protocol", can
+autonomously deploy network upgrades. Polkadot adapts to your growing needs without the risks of
+network forks.
+
+By connecting these dots, Polkadot serves as a foundational part of a decentralized web, where users
+control their data and are not limited by trust bounds within the network.
+
+## Why Polkadot?
+
+The Polkadot network is heading towards a future with a secure ecosystem for decentralized
+applications (built with blockchain technology) to communicate and interact in. Polkadot has deep
+roots in the blockchain industry and working hard to build on existing blockchain concepts like
+governance, scalibility, and many more.
+
+Back in the early 2000's, when the internet was gaining popularity for the first time, the internet
+featured read-only, static, basic webpages. The online connected world at the time was only the
+beginning of virtual data, identities, and more. The internet during this time was also called the
+Web 1.0.
+
+As social media platforms and online businesses began to emerge, the internet transformed into the
+Web 2.0. This upgraded internet, which we still use today, features dynamic, interactive webpages,
+where users can read and write information plus publish their own for others to see. This version of
+the web though, comes with downsides dealing with data control, privacy issues, and trust. This is
+where the Web 3.0 comes into the picture.
+
+The Web 3.0 is taking centralized applications and turning them into decentralized, trust-free
+protocols. The goal is to transform the internet into a decentralized web, where users control their
+own data and identity in a trust-free environment. The Web 3.0 movement aims to remove
+intermediaries and build a trustless infrastructure.
+
+> To learn more of the Web3 movement, check out this video from the
+> [Web3 Summit](https://youtu.be/l44z35vabvA)
+
+## How does Polkadot work?
+
+The Polkadot network uses a sharded model where shards - called "[parachains](learn-parachains)" -
+in the network have unique state transition functions (STF). Based on Polkadot's design, as long as
+a chain's logic can compile to Wasm, then it can connect to the Polkadot network as a parachain.
```

Comment:
```suggestion
a chain's logic can compile to Wasm and adheres to the Relay Chain API, then it can connect to the Polkadot network as a parachain.
```